common/djangoapps/config_models/test/test_utils.py

```diff
@@ -0,0 +1,63 @@
+from django.db import models
+from django.test import TestCase
+from django.utils import timezone
+
+
+from config_models.models import ConfigurationModel
+from config_models.utils import configuration_model_deserializer
+
+
+class ExampleDeserializeConfig(ConfigurationModel):
+    """
+    Test model for testing deserialization of ``ConfigurationModels`` with keyed configuration.
+    """
+    KEY_FIELDS = ('name',)
+
+    name = models.TextField()
+    int_field = models.IntegerField(default=10)
+
+
+class ConfigurationModelTests(TestCase):
+    """
+    Tests of ConfigurationModel
+    """
+    def setUp(self):
+        super(ConfigurationModelTests, self).setUp()
+
+    def test_deserialization(self):
+        self.assertEquals(0, len(ExampleDeserializeConfig.objects.all()))
+        start_date = timezone.now()
+        test_json = ('{'
```

Comment:
You could use `textwrap.dedent(""" <some multi-line-string> """)` so that you don't need to have quotes on every line.